config/application.rb

```diff
@@ -15,6 +15,7 @@
 module Convection
   class Application < Rails::Application # -- all .rb files in that directory are automatically loaded. # Application configuration should go into files in config/initializers # Settings in config/environments/* take precedence over those specified here.
     config.paths.add 'app', glob: '**/*.rb'
+    config.load_defaults 6.0
```

Comment:
for some reason when I ran app:update, it set this to 5.2. I manually updated to 6.0 since that's where we were before this update.